src/I18n/En.elm

```diff
@@ -185,6 +185,9 @@ enStrings key =
         SearchTitleFiltered num query ->
             num ++ " guides on " ++ query
 
+        SearchTitleFilteredSingleResult query ->
```

Comment:
It's better to handle the logic here so people using it don't have to remember to add logic in the view & because languages have different methods for pluralisation. https://www.gnu.org/software/gettext/manual/gettext.html#Plural-forms

Reply:
This may well be what you did in the first iteration! Sorry. Anyway, it's standard practice to pass a number and a pluralisable translation and expect the translation to handle it.

Reply:
ah! Of course, that makes more sense. Am changing.

Reply:
have also nicked the concatenation method you switched to in your PR